examples/CSharp/OpenGL Tutorials/Tutorial 1.4 - Abstractions/Texture.cs

```diff
@@ -16,72 +16,51 @@ public unsafe Texture(GL gl, string path)
 
             _handle = _gl.GenTexture();
             Bind();
-
-            //Loading an image using imagesharp.
-            using (var img = Image.Load<Rgba32>(path))
+            
+            ImageResult result = ImageResult.FromMemory(File.ReadAllBytes(path), ColorComponents.RedGreenBlueAlpha);
+            
+            fixed (byte* ptr = result.Data)
             {
-                //Reserve enough memory from the gpu for the whole image
-                gl.TexImage2D(TextureTarget.Texture2D, 0, InternalFormat.Rgba8, (uint) img.Width, (uint) img.Height, 0, PixelFormat.Rgba, PixelType.UnsignedByte, null);
-
-                img.ProcessPixelRows(accessor =>
-                {
-                    //ImageSharp 2 does not store images in contiguous memory by default, so we must send the image row by row
-                    for (int y = 0; y < accessor.Height; y++)
-                    {
-                        fixed (void* data = accessor.GetRowSpan(y))
-                        {
-                            //Loading the actual image.
-                            gl.TexSubImage2D(TextureTarget.Texture2D, 0, 0, y, (uint) accessor.Width, 1, PixelFormat.Rgba, PixelType.UnsignedByte, data);
-                        }
-                    }
-                });
+                _gl.TexImage2D(TextureTarget.Texture2D, 0, InternalFormat.Rgba, (uint) result.Width, 
+                    (uint) result.Height, 0, PixelFormat.Rgba, PixelType.UnsignedByte, ptr);
             }
 
             SetParameters();
-
         }
 
         public unsafe Texture(GL gl, Span<byte> data, uint width, uint height)
         {
-            //Saving the gl instance.
             _gl = gl;
 
-            //Generating the opengl handle;
             _handle = _gl.GenTexture();
             Bind();
 
-            //We want the ability to create a texture using data generated from code aswell.
             fixed (void* d = &data[0])
             {
-                //Setting the data of a texture.
                 _gl.TexImage2D(TextureTarget.Texture2D, 0, (int) InternalFormat.Rgba, width, height, 0, PixelFormat.Rgba, PixelType.UnsignedByte, d);
                 SetParameters();
             }
         }
 
         private void SetParameters()
         {
-            //Setting some texture perameters so the texture behaves as expected.
```

Comment:
Can we bring back comments like this one? Knowing that the texture might behave differently (or straight up not work) if you don't set these parameters would be a kind touch for those who are new to OpenGL

Reply:
I removed these comments simply because they already exist (i think?) in the textures tutorial. But there's no harm in adding them back and some people may skip to this tutorial so it seems wise to add them back.